docs/zh/latest/tutorials/protect-api.md

```diff
@@ -0,0 +1,130 @@
+---
+id: protect-api
+title: 保护 API
+keywords:
+  - API 网关
+  - Apache APISIX
+  - 发布路由
+  - 创建服务
+description: 本文介绍了如何通过 Apache APISIX 发布服务和路由。
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+本文将为你介绍使用限流限速和安全插件保护你的 API。
+
+## 概念介绍
+
+### 插件
+
+[Plugin](./terminology/plugin.md) 是扩展 APISIX 应用层能力的关键机制，也是在使用 APISIX 时最常用的资源对象，主要是在 HTTP 请求或响应生命周期期间执行的、针对请求的个性化策略。插件可以与路由、服务或消费者绑定。
+
+:::note 注意
+
+如果 [Route](./terminology/route.md)、[Service](./terminology/service.md)、[Plugin Config](./terminology/plugin-config.md) 或 Consumer 都绑定了相同的插件，则只有一份插件配置会生效，插件配置的优先级由高到低顺序是：Consumer > Route > Plugin Config > Service。同时在插件执行过程中也会涉及 6 个阶段，分别是 `rewrite`、`access`、`before_proxy`、`header_filter`、`body_filter` 和 `log`。
```

Comment:
```suggestion
如果 [路由](./terminology/route.md)、[服务](./terminology/service.md)、[插件配置](./terminology/plugin-config.md) 或消费者都绑定了相同的插件，则只有一份插件配置会生效，插件配置的优先级由高到低顺序是：消费者>路由>插件配置>服务。同时在插件执行过程中也会涉及 6 个阶段，分别是 `rewrite`、`access`、`before_proxy`、`header_filter`、`body_filter` 和 `log`。
```

Reply:
Because the relevant definitions have been explained in the previous article, I think it can be displayed in Chinese here.